index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Happy International Women’s Day</title>
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+    <section class="page-header">
+        <div class="first-header">
+            <h1>Happy International Women’s Day</h1>
+        </div>
+    
+        <section class="input">
+            <input id="addName" type="text" name="addName" placeholder="Enter name of any woman you know">
+            <button id="addNameToList"><span>Add</span></button>
+        </section>
+    
+        <section class="output">
+            <p class="output-header"> List of Awesome Women I know</p>
+                <div class id="outputList"> 
+                    <ol class id="ordered-list">
+                        <p><li>My mother</li></p>
+                        <p><li>My lady</li></p>
+                    </ol>
+                </div>
+        </section>
+    </section>
+    <script type="module" src="js/index.js"> </script>
```

Comment:
Hello @damyl4sure, is there a reason you are using `module` type here?

Reply:
Also confirm that you have the right url for the JavaScript because it only worked for me when I changed it to `src="./js/index.js"`

Reply:
I see that the use of module is not required reviewing this docs -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules 
I am removing it.

Reply:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/35523688/225373316-347069e9-33bd-4533-8bee-afec376a518c.png">